src/main.ino

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Please move this file to examples or remove it.

Reply:
Maybe the low power feature or task, should be in a different PR? maybe in the next PR after the approbation of this.

Reply:
ok good. Let me know how are the tests, to approve it.